src/validation.cpp

```diff
@@ -946,6 +961,14 @@ bool MemPoolAccept::PreChecks(ATMPArgs& args, Workspace& ws)
     }
 
     ws.m_ancestors = *ancestors;
+    // Even though just checking direct mempool parents for inheritance would be sufficient, we
+    // check using the full ancestor set here because it's more convenient to use what we have
+    // already calculated.
+    CTxMemPool::setEntries collective_ancestors = *ancestors;
+    collective_ancestors.merge(ws.m_ancestors_of_in_package_ancestors);
```

Comment:
at the introductory commit, I don't see how `m_ancestors_of_in_package_ancestors` is non-empty. unused? 

I suspect replaced by `package_with_ancestors` usage?

Reply:
Yes I think this can be scrapped now -- `ApplyV3Rules` only operates on a single transaction and its in-mempool parents; correct validation for packages should now all happen in `PackageV3Checks`.

Reply:
I fuzzed the PR with `ApplyV3Rules` turned off for `AcceptMultipleTransactions` cases and didn't hit any issues, so I believe sufficient validation is contained within `PackageV3Checks` :partying_face:

Reply:
Added "Should be called for package transactions to fail more quickly" to the doc (as in: not must call but should call).

Reply:
Or I guess we could turn it off for `AcceptMultiple`?

Reply:
If we're focusing on small packages for now, can we just disable it, rename the function something "obviously" single txn related, and then in future updates we can reintroduce the check as an optimization if we like?